### PR TITLE
CORE-15311: Added contract testing scenario for adding attachments

### PR DESCRIPTION
**Pull request by khutsijabari**
Added a test scenario to validate exception thrown when adding attachments

### CODE REVIEW

File: contracts/src/test/kotlin/com/r3/developers/csdetemplate/utxoexample/contracts/ChatContractCreateCommandTest.kt [hunk 1]
```diff
@@ -35,6 +38,34 @@ class ChatContractCreateCommandTest : ContractTest() {
...
+                .createTransactionBuilder()
+                .addAttachment(secureHash)
+                .addOutputState(outputChatState)
+                .addCommand(Create())
+                .addSignatories(outputChatState.participants)
+                .toSignedTransaction()
+        }
+        val expectedMessage = "This method is not implemented for the mock ledger"
+        val actualMessage = exception.message
+        Assertions.assertTrue(actualMessage!!.contains(expectedMessage))
```
Comment (tlawson3) [inline contracts/src/test/kotlin/com/r3/developers/csdetemplate/utxoexample/contracts/ChatContractCreateCommandTest.kt]: Can you add the static imports so you don't need to call `Assertions.assertTrue` but simply `assertTrue` like you have in the Java template?
Same goes for `Assertions.assertThrow`

**Review by tlawson3 — CHANGES_REQUESTED** [review]
Mind running the regpack against this branch and linking evidence?

**Review by yarivus — COMMENTED** [review]
pending comment from @tlawson3

**Review by yarivus — APPROVED** [review]
LGTM

File: contracts/src/test/kotlin/com/r3/developers/csdetemplate/utxoexample/contracts/ChatContractCreateCommandTest.kt (hunk 1 above)
Comment (khutsijabari) [reply contracts/src/test/kotlin/com/r3/developers/csdetemplate/utxoexample/contracts/ChatContractCreateCommandTest.kt]: Regpack test evidence:
image

**Review by tlawson3 — APPROVED** [review]

**Pull request merged**